spec/integration/billing_reporter_spec.rb

```diff
@@ -8,8 +8,9 @@
           dest_dir: '.',
           year: 2020,
           month: 1,
-          auths_json: 'spec/fixtures/total_monthly_auths.json',
           sp_yml: 'spec/fixtures/service_providers.yml',
+          auths_json: 'spec/fixtures/total_monthly_auths.json',
+          active_users_json: 'spec/fixtures/total_monthly_auths.json',
```

Comment:
❤️ for a future PR can we also add CircleCI configs?

Reply:
Sure.